tfhe/src/shortint/client_key/mod.rs

```diff
@@ -149,6 +150,49 @@ impl ClientKey {
         })
     }
 
+    /// Encrypt a small integer message using the client key with a specific message and carry
+    /// moduli
+    ///
+    /// # Warning
+    /// Defining specific message AND carry moduli might lead to undefined behavior.
```

Comment:
undefined behavior or crash ? You have an assert in the engine implem.

Reply:
Undefined behaviour as a specific meaning in C/C++/Rust,etc.

And In Rust, under no circumstance ever, a safe function (so a function not marked as `unsafe`) should
be able to invoke undefined behavior.

So either the use of 'undefined behavior' wording is not correct, or codes needs changes